core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
-- Proper location for this message, sorry for noise --

PR is coming with this fix.  I did review my JavaBeanTester code to see what was otherwise causing failure.  It was not internals for testing no-arg constructor.  It was actually kicking off a bug.  So I filed an issue to self there and will ignore related test case.